test/integration/src/test/java/io/pravega/test/integration/KeyValueTableTest.java

```diff
@@ -63,11 +118,19 @@
     private TestingServer zkTestServer = null;
     private ControllerWrapper controllerWrapper = null;
     private Controller controller;
+    private ControllerServiceImpl controllerService;
+    private RequestTracker requestTracker = new RequestTracker(true);
     private KeyValueTableFactory keyValueTableFactory;
     private final int controllerPort = TestUtils.getAvailableListenPort();
     private final String serviceHost = ENDPOINT;
     private final int servicePort = TestUtils.getAvailableListenPort();
     private final int containerCount = 4;
+    private final int startKeyRange = 10000;
+    private final int endKeyRange = 99999;
+    private final int valueSize = 10312;
+    private ScheduledExecutorService requestPool = ExecutorServiceHelpers.newScheduledThreadPool(5,
```

Comment:
Remove this. Use `executorService()` to get access to the Thread pool.

Reply:
If only I use this separate pool of threads then the Concurrent create request for same KVTable test works fine.